src/test/java/cmput402/tdd/RecipeTest.java

```diff
@@ -1,12 +1,175 @@
 package cmput402.tdd;
 
 
+import junit.framework.TestCase;
 import org.junit.Test;
 
-public class RecipeTest {
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
+public class RecipeTest extends TestCase {
     @Test
-    public void testRecipe() {
+    public void testRecipeName() {
+        //Test 1: Check initial name on creation of list
+        Recipe recipe = new Recipe("recipe1");
+        assertTrue(recipe.getName().equals("recipe1"));
 
     }
+    @Test
+    public void testSetRecipeName() {
+        //Test 2: Rename "recipie1" to "recipeRenamed"
+        Recipe recipe = new Recipe("recipe1");
+        recipe.setName("recipeRenamed");
+        assertTrue(recipe.getName().equals("recipeRenamed"));
+    }
+    @Test
+    public void testAddRecipeItem() {
+
+        Boolean status;
+        Recipe recipe = new Recipe("Apple Pie");
+        Item item1 = new Item("apple",1.0f);
```

Comment:
You should mock Item instead of creating a new Item

Reply:
Changed implementation to be mocking classes instead of instantiation